bin/grep-applab-source

```diff
@@ -48,7 +48,8 @@ puts "Fetching a list of applab apps from Pegasus DB. This step may take approxi
 puts "to run over ~3M rows in storage_apps in production..."
 
 # An array of pairs of storage_owner_ids and channel_ids, each representing an applab app.
-applab_channels = PEGASUS_DB[:storage_apps].grep(:value, '%applab%').map do |row|
+PEGASUS_REPORTING_DB = sequel_connect CDO.pegasus_reporting_db_writer, CDO.pegasus_reporting_db_reader
```

Comment:
Sorry that this wasn't in my offline conversation, I didn't glance long enough at other code... Any reason this can't be `PEGASUS_REPORTING_DB_READONLY = sequel_connect CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader`, following the pattern of (e.g.) [here](https://github.com/code-dot-org/code-dot-org/blob/ebfa56937014e79cd61e5a873b6399535d2544f4/lib/analyze_hoc_activity_helper.rb#L9).